mobile/src/lib/init.js

```diff
@@ -1,7 +1,23 @@
 import { configureReporter } from './reporter'
-import { initClient } from './cozy-helper'
+import { initClient, initBar, isClientRegistered } from './cozy-helper'
+import { revokeClient } from '../actions/authorization'
+import { startReplication } from '../actions/settings'
 
-export const initService = (store) => {
+export const initServices = (store) => {
   configureReporter(store.getState)
   initClient(store.getState().mobile.settings.serverUrl)
+
+  const client = store.getState().settings.client
+  if (client) {
+    isClientRegistered(client)
+      .then((clientIsRegistered) => {
+        if (clientIsRegistered) {
+          startReplication(store.dispatch, store.getState) // don't like to pass `store.dispatch` and `store.getState` as parameters, big coupling
```

Comment:
What do you think about this comment? Do you think it's a thing or may I remove it?